impl/src/main/java/eu/darkbot/impl/galaxy/GalaxyInfoImpl.java

```diff
@@ -81,7 +81,7 @@ private void updateGates(NodeList gates) {
 
     private void updateMultipliers(Element e) {
         XmlUtils.streamOf(e.getElementsByTagName("multiplier"))
-                .forEach(multiplier -> Optional.ofNullable(multiplier.getAttribute("mode"))
+                .forEach(multiplier -> Optional.of(multiplier.getAttribute("mode"))
                         .map(GalaxyGate::of)
                         .ifPresent(galaxyGate -> getGateInfo(galaxyGate).setMultiplier(XmlUtils.attrToInt(multiplier, "value"))));
```

Comment:
If multiplier.getAttribute can't be null, you can call .of directly instead of wrapping it in an Optional.of